Library/Formula/nodebrew.rb

```diff
@@ -22,6 +22,6 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert shell_output("#{bin}/nodebrew ls-remote").include?("v0.10.0")
+    assert_match version.to_s, shell_output("#{bin}/nodebrew --version | egrep \"nodebrew \\d\"")
```

Comment:
> But, what is v0.10.0?

Don't use Nodebrew personally but the naming of `ls-remote` would suggest its purpose is to list all the Nodes available for installation, and back in 2013 or so there was a Node `v0.10.0` release which this presumably matches.